cmd/syncEnumerator.go

```diff
@@ -51,8 +51,8 @@ func newSyncDownloadEnumerator(cca *cookedSyncCmdArgs) (enumerator *syncEnumerat
 	}
 
 	transferScheduler := newSyncTransferProcessor(cca, NumOfFilesPerDispatchJobPart)
-	includeFilters := buildIncludeFilters(cca.include, false)
-	includePathFilters := buildIncludeFilters(cca.includePath, true)
+	includeFilters := buildIncludeFilters(cca.include)
+	includePathFilters := buildIncludeFilters(cca.includePath)
```

Comment:
As per other comment, is this meant to be still here?

Reply:
Oh. Yikes. I'm gonna have to implement this in sync.

Reply:
As discussed, we're dropping include and exclude path from sync.